internal/auth0/client/resource.go

```diff
@@ -1012,6 +1012,30 @@ func NewResource() *schema.Resource {
 								},
 							},
 						},
+						"sharepoint": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							MaxItems:    1,
+							Description: "SharePoint SSO configuration.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"url": {
+										Description: "Internal SharePoint application URL.",
+										Type:        schema.TypeString,
+										Optional:    true,
```

Comment:
No URL validation needed?

Reply:
They're not present in api2 on the json schema. If needed we'll rely on the api validation directly.